content/docs/inflation-and-currency-conversions/_index.html

```diff
@@ -0,0 +1,124 @@
+---
```

Comment:
I think this file, and any other .html files can be deleted. They were auto-created by blogdown, but we only need the .md files.